### DISCUSSION TITLE
How do you all deal with app being closed after adding a job

### DISCUSSION THREAD

**Discussion by Richard-Dufour** [shinyorg/shiny#1457] [Q&A]
I realize it's kind of a weird/vague generic question :-) but how would you deal with the following scenario: 

Your MAUI app adds a job on the job manager, then the user closes the app.  Now, maybe the job started before the app got closed, but maybe it didn't. 

When the app is launched again, I guess I could check if the job was initiated/started (IsRunning, JobStarted, JobFinished) and deal with it then. If the Job has not started yet, instruct the JobManager to run it (again). 

But what if it had started before the app got closed? My app needs to do a few things upon completion of the job.
Is there an example somewhere on how to check if a job has completed? 

Ideas?

**Comment by aritchie** [comment]
That is your implementation details to sort out.  There is nothing in the framework that will do this.  You can set some flag at the end of your job to check.
